.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 6
+  },
+  "plugins": [
+    "eslint-plugin-github"
+  ],
+  "env": {
+    "es6": true,
```

Comment:
We could probably set `env.es6` `true` in https://github.com/github/eslint-plugin-github/blob/master/lib/configs/es6.js